blocks/cards/cards.css

```diff
@@ -1,27 +1,91 @@
-.cards > ul {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(257px, 1fr));
-  grid-gap: 24px;
+.cards.suggested-videos {
+  margin: 20px 0;
 }
 
-.cards > ul > li {
-  border: 1px solid #dadada;
-  background-color: var(--background-color);
+.cards.suggested-videos h3 {
+  font-family: var(--body-font-family);
+  font-size: var(--body-font-size-m);
+  font-weight: var(--font-weight-medium);
+  color: inherit;
+  margin: 0 0 -2em;
 }
 
-.cards .cards-card-body {
-  margin: 16px;
+.cards.suggested-videos .card-wrapper{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 }
 
-.cards .cards-card-image {
-  line-height: 0;
+.cards.suggested-videos .card.block {
+  width: 100%;
+  border-top: 1px solid var(--light-grey);
+  padding: 25px 0;
 }
 
-.cards > ul > li img {
-  width: 100%;
-  aspect-ratio: 4 / 3;
+.cards.suggested-videos .card.block:last-of-type {
+  border-bottom: 1px solid var(--light-grey);
+}
+
+.cards.suggested-videos .card.block>div {
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+}
+
+.cards.suggested-videos .card img::before {
+  display: inline-block;
+  width: auto;
+  height: auto;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  color: #888;
+  line-height: 1;
+  z-index: 2;
+  font-size: 2em;
+  margin-top: -25px;
+  margin-left: -15px;
+}
+
+.cards.suggested-videos .card > a {
+  display: block;
+  pointer-events: none;
+  text-align: center;
+}
+
+.cards.suggested-videos .card > a picture {
+  width: 80px;
+  height: 80px;
+  margin: 0 auto;
+}
+
+.cards.suggested-videos .card>a img {
+  border-radius: 50%;
+  position: relative;
+  left: 0;
+  width: 80px;
+  height: 80px;
   object-fit: cover;
 }
+
+.cards.suggested-videos .card .card-body {
+  display: flex;
+  position: relative;
+  top: 50%;
+  transform: translate(5%, 50%);
```

Comment:
Remove top: 50% and transform: (lines 75 + 76)... the gap I said to add plus the removal of that negative margin takes care of that.